packages/binaryen/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/WebAssembly/binaryen
+TERMUX_PKG_DESCRIPTION="Binaryen is a compiler and toolchain infrastructure library for WebAssembly"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_VERSION=version_98
```

Comment:
Please change to just 98 (and update the url to be `version_${TERMUX_PKG_VERSION}`)

Reply:
done